Added lumina-config man page. Issue #35

a-stjohn opened the pull request.

a-stjohn committed:
Added lumina-config man page. Issue #35

Mrt134 commented on manpages/lumina-config.8:
```diff
@@ -1 +1,25 @@
+
+.Sh NAME
+.Nm lumina-config
+.Nd used to modify different desktop settings.
```
"used to" is unnecessary - make the sentence active: "modifies desktop settings"

Mrt134 commented on manpages/lumina-config.8:
```diff
@@ -1 +1,25 @@
+
+.Sh DESCRIPTION
+.Nm
+opens the graphical utility to modify many different desktop settings.
```
"many different" is unnecessary. "desktop settings" implies there are different options.

Mrt134 commented on manpages/lumina-config.8:
```diff
@@ -1 +1,25 @@
+.Nm
+opens the graphical utility to modify many different desktop settings.
+Settings include appearance, desktop defaults, interface
+configuration, system settings, and user settings.
```
Suggested rework to put the list first, as it is the most important element of the sentence: "Appearance, desktop defaults, interface configuration, system settings, and user settings are adjustable." Lumina config has a large number of smaller adjustable elements - we could expand this with a list of every gui choice and a short description of the settings.

a-stjohn committed:
Fixed change requests for lumina-config. Added list of configurables.

Mrt134 committed:
Alter list appearance:

Getting a list to format the way you want is tricky. I reworked the syntax and (hopefully) provided a decent example of the different ways you can format lists.

Mrt134 commented:
Good job!
